src/main/java/com/internet/shop/util/HashUtil.java

```diff
@@ -25,7 +25,7 @@ public static String hashPassword(String password, byte[] salt) {
             }
         } catch (NoSuchAlgorithmException e) {
             System.out.println("Invalid hashing algorithm " + HASHING_ALGORITHM);
-            e.printStackTrace();
+            throw new RuntimeException();
```

Comment:
let's put an informative message here and also pass the original exception (`e`)